src/components/v2/Layout/Sidebar/Sidebar.tsx

```diff
@@ -1,77 +1,46 @@
-import React from 'react';
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
 import Toolbar from '@mui/material/Toolbar';
 import { NavLink } from 'react-router-dom';
 import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
 import ListItemIcon from '@mui/material/ListItemIcon';
 import Drawer from '@mui/material/Drawer';
-import { uid } from 'react-uid';
 import Typography from '@mui/material/Typography';
-import { ReactComponent as XVSIcon } from '../../../../assets/img/xvs.svg';
-import { ReactComponent as Logo } from '../assets/logo.svg';
-import { useStyles } from './SidebarStyles';
-
-const menuItems = [
-  {
-    href: '/dashboard',
-    text: 'Dashboard',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/vote',
-    text: 'Vote',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/xvs',
-    text: 'XVS',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/market',
-    text: 'Market',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/vault',
-    text: 'Vault',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/transaction',
-    text: 'Transaction history',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-  {
-    href: '/dev',
-    text: 'Dev',
-    icon: <XVSIcon width={20} height={20} />,
-  },
-];
+import { ReactComponent as LogoDesktop } from 'assets/img/v2/venusLogoWithText.svg';
+import { ReactComponent as LogoMobile } from 'assets/img/v2/venusLogoPure.svg';
+import { TextButton } from '../../Button';
+import { Icon } from '../../Icon';
+import { useStyles } from './styles';
+import { menuItems } from '../constants';
 
 export const Sidebar = () => {
-  const styles = useStyles();
+  const [expanded, setExpanded] = useState(true);
+  const styles = useStyles({ expanded });
   return (
-    <Drawer variant="permanent" anchor="left">
-      <Toolbar sx={styles.toolbar}>
-        <NavLink to="/">
-          <Logo />
-        </NavLink>
+    <Drawer variant="permanent" anchor="left" css={styles.drawer}>
+      <Toolbar css={styles.toolbar}>
+        <TextButton onClick={() => setExpanded(!expanded)}>
+          <LogoDesktop css={styles.logo} height="33" />
+          <LogoMobile css={styles.logoMobile} height="33" />
```

Comment:
The heights should be assigned through css, so we only have one source of truth for styles.
I know the `Icon` component uses a `size` property, that's because when I built it I had Styled Components in mind that let you assign props directly from the stylesheet (so that only one source of truth is kept). We could change that if we want too.